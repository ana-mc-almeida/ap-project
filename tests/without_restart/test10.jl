include("./common.jl")

println(to_escape() do exit
    handling(DivisionByZero =>
        (c) -> println("I saw it too")) do
        handling(DivisionByZero =>
            (c) -> (println("I saw a division by zero");
            exit("Done"))) do
            print(reciprocal(0))
        end
    end
end)
