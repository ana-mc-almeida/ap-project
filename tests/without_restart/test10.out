I saw a division by zero
Done